docs/src/app/components/pages/components/RadioButton/ExampleSimple.jsx

```diff
@@ -30,6 +32,13 @@ const RadioButtonExampleSimple = () => (
         disabled={true}
         style={styles.radioButton}
       />
+      <RadioButton
+        value="ludicrous"
```

Comment:
> I was thinking it might not be a bad idea.

I thought it was a mistake. 

> Maybe I should add a disabled checked one

What about add it to the second `RadioButtonGroup`?